pkg/agent/watch.go

```diff
@@ -230,11 +245,24 @@ func makeVMCPUMetrics(vm *vmapi.VirtualMachine) []vmMetric {
 		}
 	}
 
+	// metrics from status
 	if vm.Status.CPUs != nil {
 		m := makeVMMetric(vm, vmResourceValueStatusUse, vm.Status.CPUs.AsFloat64())
 		metrics = append(metrics, m)
 	}
 
+	// metrics from autoscaling bounds annotation
+	if bounds := extractAutoScalingBounds(vm); bounds != nil {
+		boundPairs := []pair[vmResourceValueType, resource.Quantity]{
+			{vmResourceValueAutoscalingMin, bounds.Min.CPU},
+			{vmResourceValueAutoscalingMax, bounds.Max.CPU},
+		}
+		for _, p := range boundPairs {
+			m := makeVMMetric(vm, p.first, vmapi.MilliCPUFromResourceQuantity(p.second).AsFloat64())
```

Comment:
worth explaining why we're converting to `vmapi.MilliCPU` first instead of directly converting `resource.Quantity` to float

Reply:
I wrote it this way to keep it consistent with the other CPU metrics. Do you have another reason in mind?

Reply:
oh, I think that's how it should be - I'm just saying that there should be a comment explaining why to do it this way (because `resource.Quantity.AsApproximateFloat64()` is quite inaccurate)

Reply:
https://github.com/neondatabase/autoscaling/blob/90e79ff655aac6774fa806e7928c4a3f68bbd273/neonvm/apis/neonvm/v1/virtualmachine_types.go#L200-L206

Reply:
fixed in 4ff769668e14024cee2b8679e79a7cd12c149664